### PR TITLE
Added Cypress Tests for LoginPage,Welcome Modal and Routes (Before and After Login)

### PR DESCRIPTION
**Pull request by Jonsy13**
This PR contains Cypress E2E Tests for --

- Login page

- Welcome Modal

- Routes ( Before and After Login )

- Accessibility of Community Page

- Accessibility of Workflow Starting Page

**For Now, Stubbed Server is used [ API endpoints are stubbed as required  ].**

Fixes: litmuschaos/litmus#1756

### CODE REVIEW

**Comment by Jonsy13** [comment]
Screenshot from 2020-07-21 23-47-20

File: CypressE2E/cypress/integration/Login/login.spec.js [hunk 1]
```diff
@@ -3,9 +3,52 @@
...
+})
+
+describe("Checking functionality of Login Page",()=>{
+   
+    beforeEach("Visit Login Page",()=>{
+        cy.visit('/login');
+    })
+  
+    it("Checking Input areas functionallity",()=>{
+        cy.loginInput("Vedant","Litmus");
```
Comment (uditgaurav) [inline CypressE2E/cypress/integration/Login/login.spec.js:21]: credentials can be environment variables in place of displaying here?

File: CypressE2E/cypress/integration/WelcomeModal/WelomeModal.spec.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/// <reference types="Cypress" />
+
+describe("Testing the Rechability of Welcome Modal",()=>{
```
Comment (uditgaurav) [inline CypressE2E/cypress/integration/WelcomeModal/WelomeModal.spec.js:3]: typo

Comment (Jonsy13) [reply CypressE2E/cypress/integration/WelcomeModal/WelomeModal.spec.js]: Fixed.

**Pull request merged**